### DISCUSSION TITLE
Ask for ID

### DISCUSSION THREAD

**Discussion by lkyxuan** [maximevaillancourt/digital-garden-jekyll-template#171] [Q&A] [answered]
I would like to know about the role of ID, there is an ID:Home in INDEX
![image](https://github.com/maximevaillancourt/digital-garden-jekyll-template/assets/80798796/312e00d7-2276-4591-a542-fbd1133fdb3e)

**Comment by maximevaillancourt** [comment, marked as answer]
It's not strictly needed. I'm only using it to change the page title based on whether we're on the home page (vs any other page):

https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/e2b75ba1deb979f388da7cec29e080ca56b51146/_includes/head.html#L42
